### DISCUSSION TITLE
Timeline for a new dracut release

### DISCUSSION THREAD

**Discussion by Cornelicorn** [dracutdevs/dracut#2584] [General]
Is there any estimate on when the next dracut release might happen?
The last Release is nearly a year ago, after which some nice features have been merged (e.g. new kernel install scripts).
Also some systemd-compatability stuff like #2535 or #2526.

**Comment by LaszloGombos** [comment]
There is a [commit](https://github.com/dracutdevs/dracut/commit/856e7acdb1462803c2517c8d64afb2e34c73c735) that followed the [release process](https://github.com/dracutdevs/dracut/blob/master/docs/RELEASE.md), but not (yet) tagged.

Some Linux distributions, such as Debian, Gentoo, Alpine has adopted this commit as v060.
